it/v3/endpoints/RetrieveSelfEmploymentBsasControllerISpec.scala

```diff
@@ -93,15 +112,12 @@ class RetrieveSelfEmploymentBsasControllerISpec extends IntegrationBaseSpec {
     "return error according to spec" when {
 
       def validationErrorTest(requestNino: String, requestBsasId: String, expectedStatus: Int, expectedBody: MtdError): Unit = {
-        s"validation fails with ${expectedBody.code} error" in new Test {
+        s"validation fails with ${expectedBody.code} error" in new TysIfsTest {
```

Comment:
Would be good to test for the additional tax year param errors we get for TYS here (i.e. `TaxYearFormatError`, `nvalidTaxYearParameterError` etc)